kebechet/managers/update/update.py

```diff
@@ -89,7 +89,6 @@ def repo(self):
     def repo(self, repo: git.Repo):
         """Set repository information and all derived information needed."""
         self._repo = repo
-        self.slug = repo.remote().url.split(":", maxsplit=1)[1][: -len(".git")]
```

Comment:
Was the logic behind `repo.remote()` changed? I would expect something like `git@github.com:thoth-station/adviser` reported.

Reply:
I'm missing probably context why links suddenly stopped working.

Reply:
So if you look at this - https://github.com/thoth-station/user-api/issues/1157, the Pipfile links point to the modified url from the repo link, but with gitpython change, we use temp access token to clone urls, this needed to be removed. 
In this PR I removed it from manager.py - https://github.com/thoth-station/kebechet/pull/594/files

I am not sure why there were there in setter methods, as we get the slug when manager class is initialized.

Reply:
Repo remote now is  - https://github.com/thoth-station/kebechet/blob/c232d45cf3baa10582a4ae95ae02a85b9c7fd8fc/kebechet/utils.py#L65

Reply:
OK, thanks :+1: